src/components/DutiesCard.js

```diff
@@ -0,0 +1,13 @@
+import React, {useState} from 'react';
+import OverviewCard from './OverviewCard'; //should we import Card as well?
+import '../index.css';
+
+function DutiesCard() {
+    return (
+        <div>
+            <OverviewCard/>
+        </div>
+    )
+}
```

Comment:
I'm guessing this is not quite finished?
No reason to make a component where you just wrap another component in an unnamed classless div.